src/components/m-icon/icons/core/menu-apps-icon.svg

```diff
@@ -0,0 +1,6 @@
+<svg id="layer1" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+  <rect id="rect10" width="6.667" height="6.667" fill="#4b4b4b"/>
+  <rect id="rect10-1" width="6.667" height="6.667" transform="translate(9.333)" fill="#4b4b4b"/>
+  <rect id="rect10-6" width="6.667" height="6.667" transform="translate(0 9.333)" fill="#4b4b4b"/>
+  <rect id="rect10-1-6" width="6.667" height="6.667" transform="translate(9.333 9.333)" fill="#4b4b4b"/>
+</svg>
```

Comment:
Acho que os `rect` devem receber um `fill="currentColor"` para receber qualquer cor que for passada a ela. O que acham @brunolimawd @alissonrgalindo ?

Reply:
Normalmente utilizo o currentColor, acredito que @filipe-torres pode testar se continua funcionando corretamente se funciona corretamente.

Reply:
O design do ícone do menu de aplicativos foi definido por Manoel e acredito que a cor dele será a mesma entre os diversos aplicativos do CESAR. Utilizar `fill="currentColor"` é abrir a possibilidade de quebrar com o padrão definido no design system.

Reply:
É que esse ícone está agora dentro do folder de componente de icons, então ele pode ser usado pelo usuário em outros contextos, sendo assim eu sugeriria mudar pra usar o `currentColor`

Reply:
Feito nos últimos commits.